src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -87,6 +87,10 @@ public void setTestVerbosity(){
      * END OF LARGE FILES FOR TESTING
      */
 
+    public static final String NA12878_chr17_1k_BAM = publicTestDir + "NA12878.chr17_69k_70k.dictFix.bam";
+    public static final String NA12878_chr17_1k_CRAM = publicTestDir + "NA12878.chr17_69k_70k.dictFix.cram";
+    public static final String v37_chr17_1Mb_Reference = publicTestDir + "human_g1k_v37.chr17_1Mb.fasta";
```

Comment:
fine. the current bqsr tests use hg18 which we don't have a full copy of in the lfs